sormas-app/app/src/main/java/de/symeda/sormas/app/backend/exposure/Exposure.java

```diff
@@ -60,6 +60,8 @@
 	private EpiData epiData;
 	@DatabaseField(foreign = true, foreignAutoRefresh = true)
 	private User reportingUser;
+	@DatabaseField(dataType = DataType.BOOLEAN)
```

Comment:
Remove the dataType part (for consistency reasons, we're not really using it in other places where we define booleans in the app)